src/server/models/listItem/providers/deserialisers/types.ts

```diff
@@ -63,6 +63,7 @@ export interface BaseDeserialisedWebhookData {
   emergencyPhoneNumber?: string;
   declaration: string[];
   type: ServiceType;
+  updatedJsonData?: Omit<BaseDeserialisedWebhookData, "updatedJsonData">
```

Comment:
nice

Reply:
I don't think it was mine originally. cc @RichardBray 🙂